examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart

```diff
@@ -273,118 +273,122 @@ class Tab1ItemPageState extends State<Tab1ItemPage> {
         middle: new Text(widget.colorName),
         trailing: const ExitButton(),
       ),
-      child: new ListView(
-        children: <Widget>[
-          const Padding(padding: const EdgeInsets.only(top: 16.0)),
-          new Padding(
-            padding: const EdgeInsets.symmetric(horizontal: 16.0),
-            child: new Row(
-              mainAxisSize: MainAxisSize.max,
-              children: <Widget>[
-                new Container(
-                  height: 128.0,
-                  width: 128.0,
-                  decoration: new BoxDecoration(
-                    color: widget.color,
-                    borderRadius: new BorderRadius.circular(24.0),
+      child: new SafeArea(
```

Comment:
Do you need to add it to line 178 too?

Reply:
Nice catch. As-is, the `SliverPadding` on line 134 applies the left/right safe areas, so the layout looked good, but we failed to respond to taps in the padding area. I've removed the horizontal padding there, and applied it at line 178 instead.